flutter/cpp/flutter/main.h

```diff
@@ -38,6 +38,7 @@ struct dart_ffi_run_benchmark_out {
   char *accuracy;
   int32_t num_samples;
   float duration_ms;
+  char *backend_description;
```

Comment:
Why do we need this? I believe it would be easier to change the way we store backend name and accelerator in the `Benchmark` class (I mean, store it separately).
If anything, I would like to reduce size of the `RunResult` class. Populating `RunResult` using C++ code only increases complexity.

Reply:
It is to pass the backend name of `ExternalBackend` from C++ to dart. 
How would you get the backend name for each task?

Reply:
Oh, I see. I misinterpreted this as being the same as `benchmarkSetting.configuration`.

Well, ideally I would extract the backend name from `dart_ffi_backend_match` function, but I'm not sure how much changes it would require to implement, so maybe it's not worth it.